### PR TITLE
Add notify msg for online player group change

### PR DESCRIPTION
**Pull request by moisterrific**
Minor issue: changing a player's group permanently does not send a success msg to the target player, this is inconsistent considering `tempgroup` sends a success msg.

This makes it more consistent with how `/tempgroup` works, also added silent command support.

?????? HAVE YOU UPDATED THE CHANGELOG? ??????
yes

### CODE REVIEW

File: TShockAPI/Commands.cs [hunk 1]
```diff
@@ -1116,6 +1116,12 @@ private static void ManageUsers(CommandArgs args)
 					TShock.UserAccounts.SetUserGroup(account, args.Parameters[2]);
 					TShock.Log.ConsoleInfo(args.Player.Name + " changed account " + account.Name + " to group " + args.Parameters[2] + ".");
 					args.Player.SendSuccessMessage("Account " + account.Name + " has been changed to group " + args.Parameters[2] + "!");
+					
+					var players = TSPlayer.FindByNameOrID(account.Name);
+					if (players.Count == 0)
+						return;
+					else if (!args.Silent)
+						players[0].SendSuccessMessage($"{args.Player.Name} has changed your group to {args.Parameters[2]}");
```
Comment (QuiCM) [inline TShockAPI/Commands.cs]:
```suggestion
					//send message to player with matching account name
					var player = TShock.Players.FirstOrDefault(p => p != null && p.Account?.Name == account.Name)
					if (player != null && !args.Silent)
						player.SendSuccessMessage($"{args.Player.Name} has changed your group to {args.Parameters[2]}");
```

**Review by QuiCM — CHANGES_REQUESTED** [review]

File: TShockAPI/Commands.cs (hunk 1 above)
Comment (QuiCM) [reply TShockAPI/Commands.cs]: Pleaase note this suggestion was written by hand & untested - will need to be sanity checked

Comment (moisterrific) [reply TShockAPI/Commands.cs]: I can test it and let u know

Comment (moisterrific) [reply TShockAPI/Commands.cs]: tested it and it works, just missing a `;`

**Review by moisterrific — COMMENTED** [review]
?

**Review by hakusaro — CHANGES_REQUESTED** [review]
This is pretty good but why does the PR not have any description of what changed?

**Comment by moisterrific** [comment]
> 
> 
> This is pretty good but why does the PR not have any description of what changed?

fixed

**Review by hakusaro — DISMISSED** [review]

**Review by QuiCM — DISMISSED** [review]

**Review by QuiCM — APPROVED** [review]

**Pull request merged**